hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -1809,90 +1830,94 @@ private void deleteCheckOnParentBlob(final Path path,
         getAbfsConfiguration().getBlobDirDeleteMaxThread());
-    while (!consumer.isCompleted()) {
-      final List<BlobProperty> blobList = consumer.consume();
-      if (blobList == null) {
-        continue;
-      }
-      List<Future> futureList = new ArrayList<>();
-      for (BlobProperty blobProperty : blobList) {
-        futureList.add(deleteBlobExecutorService.submit(() -> {
-          try {
-            LOG.debug(String.format("Deleting Path %s",
-                blobProperty.getPath().toUri().getPath()));
-            client.deleteBlobPath(blobProperty.getPath(), null, tracingContext);
-            LOG.debug(String.format("Deleted Path %s",
-                blobProperty.getPath().toUri().getPath()));
-          } catch (AzureBlobFileSystemException ex) {
-            if (ex instanceof AbfsRestOperationException
-                && ((AbfsRestOperationException) ex).getStatusCode()
-                == HttpURLConnection.HTTP_NOT_FOUND) {
-              LOG.debug(String.format("Path %s already deleted",
-                  blobProperty.getPath().toUri().getPath()));
-              return;
+    try {
+      while (!consumer.isCompleted()) {
+        final List<BlobProperty> blobList = consumer.consume();
+        if (blobList == null) {
+          continue;
+        }
+        List<Future> futureList = new ArrayList<>();
+        for (BlobProperty blobProperty : blobList) {
+          futureList.add(deleteBlobExecutorService.submit(() -> {
+            String blobPropertyPathStr = blobProperty.getPath()
+                .toUri()
+                .getPath();
+            try {
+              LOG.debug(String.format("Deleting Path %s", blobPropertyPathStr));
+              client.deleteBlobPath(blobProperty.getPath(), null,
+                  tracingContext);
+              LOG.debug(String.format("Deleted Path %s", blobPropertyPathStr));
+            } catch (AzureBlobFileSystemException ex) {
+              if (ex instanceof AbfsRestOperationException
+                  && ((AbfsRestOperationException) ex).getStatusCode()
+                  == HttpURLConnection.HTTP_NOT_FOUND) {
+                LOG.debug(String.format("Path %s already deleted",
+                    blobPropertyPathStr));
+                return;
+              }
+              LOG.error(String.format("Deleting Path %s failed",
+                  blobPropertyPathStr));
+              throw new RuntimeException(ex);
             }
-            LOG.error(String.format("Deleting Path %s failed",
-                blobProperty.getPath().toUri().getPath()));
-            throw new RuntimeException(ex);
-          }
-        }));
-      }
+          }));
+        }
 
-      for (Future future : futureList) {
-        try {
-          future.get();
-        } catch (InterruptedException | ExecutionException e) {
-          deleteBlobExecutorService.shutdown();
-          throw new RuntimeException(e);
+        for (Future future : futureList) {
```

Comment:
Spacing issue

Reply:
It seems you are checking on the commit change. Kindly clarify what has to be changed here please. Thanks.

Reply:
Spacing issue from the previous commit